<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNUpgradeWidget</class>
 <widget class="QWidget" name="HNUpgradeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HNUpgradeWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetHNUpgrade" native="true">
     <widget class="QLabel" name="lbTip">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>231</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>System Upgrade:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="HNProgressBar" name="widgetUpgrade" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>611</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbUpgrade">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>103</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Wating...</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tbVersion">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>601</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDown">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HNProgressBar</class>
   <extends>QWidget</extends>
   <header>hnprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
